src/app/core/tools/handlers/CopyHandler.ts

```diff
@@ -27,7 +27,15 @@ export const CopyHandler: EventHandler = ({
         // We don't copy the data from the json since it will cause 
         // some weird error, which will cause the issue #746
         ev.clipboardData.setData("text/plain", str);
-        ev.preventDefault(); // Necessary to copy correctly
+
+        // If an editable component (LABEL) is not selected, 
+        // or if an editable component is selected but it is a "Caret" (cursor) instead of highlighted text,
```

Comment:
Definitely clearer, but where exactly did you get the string `"LABEL"`, why is it in all caps? I've never seen it before so I'm more just wondering.

Reply:
Basically I noticed that all the input elements in the popup modules had that anchor node name. I believe it's because in Module.tsx there's this section for the popup module:
```
export const PopupModule = (({label, modules}: PopupModuleProps) => {
    return function PopupModuleFunc (props: UseModuleProps) {
        const ms = modules.map(m => m(props));
        if (ms.every(m => m === null))
            return null;
        return <div key={`selection-popup-${label}-module`}>
            {label}
            <label unselectable="on">
                {ms.map((m, i) => cloneElement(m!, {key: `selection-popup-${label}-module-${i}`}))}
            </label>
        </div>;
    }
});
```
Each of the inputs seems to be surrounded by a label element, which is why when my cursor is in an input and I call the anchor node name, it returns label. I'm not sure why it's in all caps, but it seems to be the same for all other types of node names as well (I'm assuming it's just an HTML thing?).